django/contrib/admin/static/admin/js/calendar.js

```diff
@@ -8,8 +8,8 @@ depends on core.js for utility functions like removeChildren or quickElement
     'use strict';
     // CalendarNamespace -- Provides a collection of HTML calendar-related helper functions
     var CalendarNamespace = {
-        monthsOfYear: gettext('January February March April May June July August September October November December').split(' '),
-        daysOfWeek: gettext('S M T W T F S').split(' '),
+        monthsOfYear: gettext('January|February|March|April|May|June|July|August|September|October|November|December').split('|'),
```

Comment:
I think we could use an array here instead of splitting by a separator.

e.g.
monthsOfYear = [
  gettext('January'),
  gettext('February'),
  ...
  gettext('December')
]